homeassistant/components/myq/cover.py

```diff
@@ -61,6 +62,11 @@ def is_opening(self):
         """Return if the cover is opening or not."""
         return MYQ_TO_HASS.get(self._device.state) == STATE_OPENING
 
+    @property
+    def extra_state_attributes(self):
+        """Return entity specific state attributes."""
+        return {ATTR_POSITION: MYQ_TO_HASS.get(self._device.state)}
```

Comment:
What information does this provide that is not already available in the state and state attributes?

Reply:
The `secondary_info: position` for entities card is not working.

![image](https://user-images.githubusercontent.com/45110141/147883998-610eba54-ffa1-4b51-a00f-bd181737f297.png)


As an alternative I made the code change in this PR and then changed the type of the entity to `type: attribute` and then added `attribute: position` to get the card to show the status instead of up and down arrows.

![image](https://user-images.githubusercontent.com/45110141/147883980-6f7a3bd0-48f0-483d-ac3c-5f67c00ca077.png)

If there is a different approach I am happy to take it.. I tried getting the `secondary_info` to work instead but had no success doing that.  My goal is to get the status to show in some way - I am happy to go a different route if this is not ideal/optimal.

Thank you,
DeadEnd

Reply:
That is a fontend issue, which should not be solved in the backend.

Reply:
The card has enough information be able to derive the position from the state.  I would suggest opening a PR to the card to add support for displaying the state

Reply:
Okay, I will close this ticket then and proceed as suggested.

I appreciate all of your time looking at this and giving feedback!
Cheers!
DeadEnd